src/sunmatrix/sparse/sunmatrix_sparse.c

```diff
@@ -22,19 +22,21 @@
 
 #include <stdio.h>
 #include <stdlib.h>
+#include <sundials/priv/sundials_errors_impl.h>
 #include <sundials/sundials_math.h>
-#include <sundials/sundials_nvector.h>
+#include <sunmatrix/sunmatrix_dense.h>
```

Comment:
This is already in `sunmatrix_sparse.h`. if it needs to be here for include what you use, then `sunmatrix_band.h` should be here too.